inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -440,14 +442,11 @@ private void jumpToRecommendationLocationAndHighlightRecommendation(AjaxRequestT
 
     private FeatureEditor createFeatureEditor()
     {
-        if (currentRecommendation != null) {
-            annotationFeature = annotationService
-                .getFeature(currentRecommendation.getFeature(), selectedLayer.getObject());
-        }
-        else {
-            annotationFeature = annotationService
-                .listAnnotationFeature(selectedLayer.getObject()).get(0);
-        }
+        annotationFeature = annotationService
+            .getFeature(currentRecommendation.getFeature(), selectedLayer.getObject());
+
+        annotationFeature = annotationService.listAnnotationFeature(selectedLayer.getObject())
```

Comment:
Why are you assigning annotationFeature twice in a row?

Reply:
Sorry... It's a mistake caused by changing the old codes.